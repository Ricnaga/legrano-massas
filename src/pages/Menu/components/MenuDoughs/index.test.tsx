import { MenuDoughs } from '.';
import { renderWithTheme } from '../../../../application/test/testing-library';

jest.mock('./hooks/useMenuDoughs', () => ({
  __esModule: true,
  useMenuDoughs: () => ({
    data: {
      menuState: [
        {
          id: 'mock id',
          category: 'mock category',
          items: [
            {
              id: 'mock id',
              name: 'mock name',
              price: 1,
              weight: 'mock weight',
              amount: 1,
              selected: false,
            },
          ],
        },
      ],
    },
    functions: { onAddItemToCart: jest.fn() },
  }),
}));

describe('MenuDoughs component', () => {
  it('should render correctly', () => {
    const { container } = renderWithTheme(<MenuDoughs />);
    expect(container).toMatchSnapshot();
  });
});
