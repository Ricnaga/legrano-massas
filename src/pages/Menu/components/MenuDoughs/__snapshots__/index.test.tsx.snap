// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuDoughs component should render correctly 1`] = `
<div>
  <div
    class="MuiGrid-root css-vj1n65-MuiGrid-root"
  >
    <div
      class="MuiGrid-root css-vj1n65-MuiGrid-root"
    >
      <h1
        class="MuiTypography-root MuiTypography-h1 MuiTypography-alignCenter css-xv1x7b-MuiTypography-root"
      >
        mock category
      </h1>
      <ul
        class="MuiImageList-root MuiImageList-standard css-rvv586-MuiImageList-root"
        style="grid-template-columns: repeat(2, 1fr); gap: 18px;"
      >
        <li
          class="MuiImageListItem-root MuiImageListItem-standard css-ta95xa-MuiImageListItem-root"
          style="height: auto; grid-column-end: span 1; grid-row-end: span 1;"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-16979di-MuiPaper-root-MuiCard-root"
          >
            <img
              alt="Em desenvolvimento"
              aria-label="Em breve teremos imagens :)"
              class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-o69gx8-MuiCardMedia-root"
              data-mui-internal-clone-element="true"
              height="200"
              src="https://images.unsplash.com/photo-1584824486509-112e4181ff6b"
            />
          </div>
          <div
            class="MuiImageListItemBar-root MuiImageListItemBar-positionBelow css-1dkvcjw-MuiImageListItemBar-root"
          >
            <div
              class="MuiImageListItemBar-titleWrap MuiImageListItemBar-titleWrapBelow css-186gwnw-MuiImageListItemBar-titleWrap"
            >
              <div
                class="MuiImageListItemBar-title css-dasnyc-MuiImageListItemBar-title"
              >
                <div
                  class="MuiGrid-root MuiGrid-container css-9cyib4-MuiGrid-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-jrlmwg-MuiTypography-root"
                  >
                    mock name
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-jrlmwg-MuiTypography-root"
                  >
                    mock weight
                  </p>
                </div>
              </div>
              <div
                class="MuiImageListItemBar-subtitle css-hk6gwf-MuiImageListItemBar-subtitle"
              >
                <div
                  class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  >
                    <div
                      class="MuiButtonBase-root MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-clickable MuiChip-clickableColorPrimary MuiChip-filledPrimary css-1myczzk-MuiButtonBase-root-MuiChip-root"
                      role="button"
                      tabindex="0"
                    >
                      <span
                        class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
                      >
                        <div
                          class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary MuiSvgIcon-fontSizeMedium css-xcwv2-MuiSvgIcon-root"
                            data-testid="AddShoppingCartIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 9h2V6h3V4h-3V1h-2v3H8v2h3v3zm-4 9c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2zm-9.83-3.25.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.86-7.01L19.42 4h-.01l-1.1 2-2.76 5H8.53l-.13-.27L6.16 6l-.95-2-.94-2H1v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.13 0-.25-.11-.25-.25z"
                            />
                          </svg>
                          <p
                            class="MuiTypography-root MuiTypography-body2 css-7dzi2e-MuiTypography-root"
                          >
                            Colocar no carrinho
                          </p>
                        </div>
                      </span>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </div>
                  </div>
                  <div
                    class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-jrlmwg-MuiTypography-root"
                    >
                      R$Â 36,00
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
